administration/src/bp-modules/self-service/CardSelfServiceForm.tsx

```diff
@@ -43,12 +43,27 @@ const CardSelfServiceForm = ({
 }: CardSelfServiceFormProps): ReactElement => {
   const { viewportSmall } = useWindowDimensions()
   const projectConfig = useContext(ProjectConfigContext)
+  const [formSendAttempt, setFormSendAttempt] = useState(false)
+  const [touchedFullName, setTouchedFullName] = useState(false)
   const [openDataPrivacy, setOpenDataPrivacy] = useState<boolean>(false)
   const [openReferenceInformation, setOpenReferenceInformation] = useState<boolean>(false)
   const cardValid = isValid(card, { expirationDateNullable: true })
   const appToaster = useAppToaster()
 
+  const updateFullName = (fullName: string) => {
+    updateCard({ fullName })
+    if (!touchedFullName) {
+      setTouchedFullName(true)
+    }
+  }
+
   const createKoblenzPass = async () => {
+    if (!formSendAttempt) {
+      setFormSendAttempt(true)
+    }
+    if (!touchedFullName) {
+      setTouchedFullName(true)
+    }
```

Comment:
:wrench: The checks are not needed, if the state is already true before, nothing happens (as the value does not change).
Also, it should be enough to only set `formSendAttempt` to true.
```suggestion
    setFormSendAttempt(true)
```